downstream/modules/automation-hub/proc-obtain-images.adoc

```diff
@@ -0,0 +1,46 @@
+
+
+[id="obtain-images"]
+
+
+= Obtaining images for use in {HubName}
+
+
+[role="_abstract"]
+Before you can push container images to your private {HubName}, you must first pull them from an existing registry and tag them for use. This example details how to pull an image from the Red Hat Ecosystem Catalog (registry.redhat.io).
+
+.Prerequisites
+
+* You have permissions to pull images from registry.redhat.io
+
+.Procedure
+
+. Log in to Podman using your registry.redhat.io credentials:
++
+-----
+$ podman login `registry.redhat.io`
```

Comment:
I don't think the backticks should be in the command
https://access.redhat.com/RegistryAuthentication